container-runtime/podman_config.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+#
+# A little script to get needed linux packages, change podman config files, and add a small
+# kontain selinux policy to allow podman to run containers using krun and km.
+#
+
+CONTAINERS_CONF=/usr/share/containers/containers.conf
+HOME_CONTAINERS_CONF=~/.config/containers/containers.conf
+ETC_CONTAINERS_CONF=/etc/containers/containers.conf
+DOCKER_INIT=/usr/libexec/docker/docker-init
+KRUN_PATH=/opt/kontain/bin/krun
+KM_PATH=/opt/kontain/bin/km
+KM_SELINUX_CONTEXT="system_u:object_r:bin_t:s0"
+
+# Either do it or just print the commands we think should be run.
+#DEBUG=echo
+DEBUG=""
+
+# Install podman
+if ! dnf list installed | grep -q podman
+then
+   echo "podman is not installed, installing it"
+   $DEBUG sudo dnf -y install podman
+else
+   echo "podman is installed"
+fi
```

Comment:
Why not just install it? It will do nothing if already installed, and say nothing with -q `sudo dnf -q install podman`